src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -63,33 +63,30 @@ namespace winrt::TerminalApp::implementation
         _tabView = _tabRow.TabView();
         _rearranging = false;
 
-        // weak_ptr to this TerminalPage object lambda capturing
-        auto weakThis{ get_weak() };
-
-        _tabView.TabDragStarting([weakThis](auto&& /*o*/, auto&& /*a*/) {
+        _tabView.TabDragStarting([weakThis{ get_weak() }, this](auto&& /*o*/, auto&& /*a*/) {
```

Comment:
This file too... I feel better about using the actual strong pointer than by wink-nudge using the `this` simply because it turned out that the weak could be strengthened.